Cargo.toml

```diff
@@ -128,6 +128,7 @@ tracing-tower = { git = "https://github.com/tokio-rs/tracing", default-features
 tracing-opentelemetry = { version = "0.15.0", default-features = false }
 opentelemetry = { version = "0.16.0", default-features = false, features = ["trace", "rt-tokio"] }
 opentelemetry-datadog = { version = "0.4.0", default-features = false, features = ["reqwest-client"] }
+console-subscriber = { git = "https://github.com/tokio-rs/console" }
```

Comment:
Nit: should we do the same `default-features = false` here?

Also, do we still want this? I see the code that was using it is currently commented out.

Reply:
Resolved in 0f4aac626